### PR TITLE
add docs for creating a user_group

### PR DESCRIPTION
**Pull request by RobinHeathcote**

### CODE REVIEW

File: source/includes/api/core/_user_groups.md [hunk 1]
```diff
@@ -49,3 +49,53 @@ name | string | user_group name
 parent\_id | integer | parent user_group id
 default | boolean | indicates if this user_group is the default group for the programme
 position | integer | position under the parent user_group, used for ordering
+
+### Manage User Groups
+
+An endpoint to create a usergroup for this programme. Only accessible if the key
```
Comment (rjpaskin) [inline source/includes/api/core/_user_groups.md]: 'usergroup' -> 'user group'?

File: source/includes/api/core/_user_groups.md [hunk 2]
```diff
@@ -49,3 +49,53 @@ name | string | user_group name
...
+has been granted access to users. In order to create a usergroup a name is required.
+This usergroup can be nested underneath another usergroup by passing the parent
+user group id. Images can be uploaded to the usergroup by passing an image url. If
```
Comment (rjpaskin) [inline source/includes/api/core/_user_groups.md]: Specify that image URL must be publicly-accessible?

File: source/includes/api/core/_user_groups.md [hunk 3]
```diff
@@ -49,3 +49,53 @@ name | string | user_group name
...
+no position is passed then it will default to the bottom of the hierachy.
+
+``` http
+POST /api/v2/user_groups HTTP/1.1
+Authorization: Token token=xxx
+
+{
+  "name": "Example User Group",
+  "position": 1,
+  "parent_id": nil,
```
Comment (hass-laughingman) [inline source/includes/api/core/_user_groups.md]: json doesn't understand `nil` please use `null` throughout this file

File: source/includes/api/core/_user_groups.md [hunk 4]
```diff
@@ -49,3 +49,53 @@ name | string | user_group name
...
 default | boolean | indicates if this user_group is the default group for the programme
 position | integer | position under the parent user_group, used for ordering
+
+### Manage User Groups
+
+An endpoint to create a usergroup for this programme. Only accessible if the key
+has been granted access to users. In order to create a usergroup a name is required.
+This usergroup can be nested underneath another usergroup by passing the parent
+user group id. Images can be uploaded to the usergroup by passing an image url. If
+no position is passed then it will default to the bottom of the hierachy.
```
Comment (rjpaskin) [inline source/includes/api/core/_user_groups.md]: Double-check: does it not end up as a child of the default user group "users"?

Comment (RobinHeathcote) [reply source/includes/api/core/_user_groups.md]: Just checked and it just ends up at the bottom of the list ie not nested

Comment (rjpaskin) [reply source/includes/api/core/_user_groups.md]: Ok thanks for checking!

**Pull request merged**